modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,132 @@
+----
+. Generate a shared access signature (SAS) token. Use this token to upload the {op-system} VHD to your storage container with the following commands: 
++
+[source,terminal]
+----
+$ end=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
+----
++
+[source,terminal]
+----
+$ sas=`az storage container generate-sas -n ${CONTAINER_NAME} --account-name ${STORAGE_ACCOUNT_NAME} --https-only --permissions dlrw --expiry $end -o tsv`
+----
+. Copy the {op-system} VHD into the storage container:
++
+[source, terminal]
+----
+$ az storage blob copy start --account-name ${STORAGE_ACCOUNT_NAME} --sas-token "$sas" \
+ --source-uri "${RHCOS_VHD_ORIGIN_URL}" \
+ --destination-blob "${BLOB_NAME}" --destination-container ${CONTAINER_NAME}
+----
++
+You can check the status of the copying process with the following command:
++
+[source,terminal]
+----
+$ az storage blob show -c ${CONTAINER_NAME} -n ${BLOB_NAME} --account-name ${STORAGE_ACCOUNT_NAME} | jq .properties.copy
+----
++
+.Example output
+[source,terminal]
+----
+{
+ "completionTime": null,
+ "destinationSnapshot": null,
+ "id": "1fd97630-03ca-489a-8c4e-cfe839c9627d",
+ "incrementalCopy": null,
+ "progress": "17179869696/17179869696",
+ "source": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-411.86.202207130959-0-azure.aarch64.vhd",
+ "status": "success", <1>
+ "statusDescription": null
+}
+----
++
+<1> If the status parameter displays the `success` object, the copying process is complete. 
+ 
+. Create an image gallery using the following command:
++
+[source,terminal]
+----
+$ az sig create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME}
+----
+Use the image gallery to create an image definition. In the following example command, `rhcos-arm64` is the name of the image definition. 
++
+[source,terminal]
+----
+$ az sig image-definition create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME} --gallery-image-definition rhcos-arm64 --publisher RedHat --offer arm --sku arm64 --os-type linux --architecture Arm64 --hyper-v-generation V2
+----
+. To get the URL of the VHD and set it to `RHCOS_VHD_URL` as the file name, run the following command: 
++
+[source,terminal]
+----
+$ RHCOS_VHD_URL=$(az storage blob url --account-name ${STORAGE_ACCOUNT_NAME} -c ${CONTAINER_NAME} -n "${BLOB_NAME}" -o tsv)
+----
+. Use the `RHCOS_VHD_URL` file, your storage account, resource group, and image gallery to create an image version. In the following example, `1.0.0` is the image version.
++
+[source,terminal]
+----
+$ az sig image-version create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME} --gallery-image-definition rhcos-arm64 --gallery-image-version 1.0.0 --os-vhd-storage-account ${STORAGE_ACCOUNT_NAME} --os-vhd-uri ${RHCOS_VHD_URL}
+----
+. Your `arm64` image ID for the `resourceID` parameter is now generated. You can access your `resourceID` with the following command: 
```

Comment:
instead of `Your `arm64` image ID for the `resourceID` parameter is now generated` , maybe `The arm64 bootimage is created. You can access the id of the image with the following command` ?

might want to call out that the `id` field in the json is what is used for `resourceID` in the machineset

Reply:
Acked, updating